sycl/plugins/esimd_emulator/pi_esimd_emulator.cpp

```diff
@@ -263,8 +263,8 @@ template <int DIMS> class libCMBatch {
     const auto InvokeKernelArg = KernelInvocationContext<DIMS>{
         MKernel, LocalSize, GlobalSize, GlobalOffset};
 
-    EsimdemuKernel{reinterpret_cast<fptrVoid>(InvokeKernel<DIMS>), GroupDim,
-                   SpaceDim}
+    EsimdemuKernel{reinterpret_cast<fptrVoid>(InvokeKernel<DIMS>),
+                   GroupDim.data(), SpaceDim.data()}
```

Comment:
I think the new interface should then include array length, otherwise you can't guess it within EsimdemuKernel constructor code

Reply:
CM_EMU library will be updated to have size argument for dimension info. I'll post a PR for updated interface when updated library package is posted in opensource CM_EMU github repo.